app/public/stylesheets/sass/_components/_view-container.scss

```diff
@@ -34,4 +34,9 @@
     &:last-child {
       margin-right: 0;
     }
-}
\ No newline at end of file
+}
+
+.streamView_inner {
+  margin-top: 50px;
+  margin-bottom: -50px;
+}
```

Comment:
oops. i must have accidentally left that in. I was testing to see if that did anything to the second row.